frontend/src/components/eMIB/OrganizationalStructure.jsx

```diff
@@ -55,39 +43,26 @@ class OrganizationalStructure extends Component {
     this.setState({ showPopupBox: false });
   };
 
-  // loads the markdown and tree view content (english and french versions)
-  componentDidMount = () => {
-    this.props.getTestQuestions(TEST_DEFINITION.emib.sampleTest).then(response => {
-      // Save the organizational structure markdown and tree view content in local states.
-      this.setState({
-        markdown_en: response.background.en.background[0].markdown[2].text,
-        markdown_fr: response.background.fr.background[0].markdown[2].text,
-        popupMarkdownTitle_en: response.background.en.background[0].markdown[5].text,
-        popupMarkdownTitle_fr: response.background.fr.background[0].markdown[5].text,
-        popupMarkdownDescription_en: response.background.en.background[0].markdown[6].text,
-        popupMarkdownDescription_fr: response.background.fr.background[0].markdown[6].text,
-        treeViewContent_en:
-          response.background.en.background[0].tree_view[0].organizational_structure_tree_child,
-        treeViewContent_fr:
-          response.background.fr.background[0].tree_view[0].organizational_structure_tree_child,
-        isLoadingComplete: true
-      });
-    });
-  };
-
   render() {
-    const { currentLanguage } = this.props;
-    const { treeViewContent_en, treeViewContent_fr } = this.state;
-    let treeView = [];
-    // waiting for tree view content data loading
-    if (this.state.isLoadingComplete) {
-      treeView = processTreeContent(
-        currentLanguage,
-        treeViewContent_en,
-        treeViewContent_fr,
-        "organizational_structure_tree_child"
-      );
-    }
+    const { currentLanguage, testBackground } = this.props;
+
+    const treeViewContent_en =
+      testBackground.en.background[0].tree_view[0].organizational_structure_tree_child;
+    const treeViewContent_fr =
+      testBackground.fr.background[0].tree_view[0].organizational_structure_tree_child;
+    const markdown_en = testBackground.en.background[0].markdown[2].text;
+    const markdown_fr = testBackground.fr.background[0].markdown[2].text;
```

Comment:
we may need to find a more dynamic (less hardcoded) way to load these for the pizza test: the hard coded order might be the same in pizza or other future tests. Then again it might be. Whatever the case, out of scope for this PR

Reply:
We definitely need to - it's currently the error I get when I try loading the pizza test. This is a logged item in the pizza trello card.